test-case/manual/check-simple-probe-capture.md

```diff
@@ -0,0 +1,78 @@
+# Probes test
+Simple probes functionality test.
+
+# Preconditions
+1. Linux kernel should have these optios enabled:
```

Comment:
Typo: options
but 'option' is confusing to me, should we say CONFIG

Reply:
Thanks, will apply.